GDJS/Runtime/runtimeobject.js

```diff
@@ -1145,7 +1208,7 @@ gdjs.RuntimeObject.prototype.separateFromObjectsList = function(objectsLists, ig
 
             //Check if their is a collision with each object
             for(var i = 0, len = objects.length;i<len;++i) {
-                if ( objects[i].id != this.id ) {
+                if ( objects[i].id != this.getUniqueId() ) {
```

Comment:
`id` was not renamed here. But as I said, I strongly strongly suggest you don't change this now. 

Let's go step by step ;) Safety first with types. Refactoring can be done later.
Users won't care if id is called `_id` or `id`, but they will care about a regression.